README.md

```diff
@@ -85,16 +96,15 @@ When working with the [test data](/resources/video_game_reviews_example.json) a
 - The path to the file should be configurable
 - Keep in mind that the file might be too large to fit into memory
 
-## Running your service
+## Running the service
 
 We will run your service and see if it does what it is expected to do.
-Please make sure to give us some instructions about how we can run it so that it works for us.
+Please make sure to give us some instructions about how we can run it on our machines.
 Ideally it should run independent on the platform and not require a big machine.
 
 ## Tips
-
-- You should solve this exercise as if you would solve a problem at work, writing code that will run in production.
-- Parts of this exercise and its requirements are unclear, left out or ambiguous on purpose. We want to see how you tackle that and also want to allow a variety of solutions.
+- If you are getting stuck and unable to proceed with the exercise, or if you have any other questions, don't hesitate to contact us. Communication is key.
 - Please explicitly note any assumptions that you make.
-- The choice of libraries and framework is on purpose completely up to you. But keep in mind that at Hivemind we place a high value on purely functional programming and would appreciate your code adhering to that.  
-- If you are getting stuck and unable to proceed with the exercise, or if you have any other questions, don't hesitate to contact us.
+- The choice of libraries and framework is on purpose completely up to you. 
```

Comment:
Is it worth adding that these decisions will be discussed at the next stage?
Something like: 
```suggestion
- The choice of libraries and framework is on purpose completely up to you, and these decisions can form a basis of discussion for any further interview.
```